prisma/schema.prisma

```diff
@@ -112,18 +103,114 @@ model Area {
   process_practices    String[]
   technology_practices String[]
   roles                String[]
-  question             String?
+  questions            Question[]
```

Comment:
Brought back the `question` field as it is a valid entry in the Area models in the Refa, we don't use it as it is scarce, but it needs to be there. Renamed `questions` to `assessment_questions`